libeventheader-tracepoint/include/eventheader/TraceLoggingProvider.h

```diff
@@ -1185,6 +1189,14 @@ Notes on serializing data:
 #endif // __cplusplus
 #endif // _tlg_NOEXCEPT
 
+#ifndef _tlg_STATIC_ASSERT
+#if defined(static_assert) || defined(__cplusplus)
+#define _tlg_STATIC_ASSERT(cond, msg) static_assert(cond, msg)
+#else
+#define _tlg_STATIC_ASSERT(cond, msg)
+#endif
+#endif // _tlg_STATIC_ASSERT
```

Comment:
Note: Define a local STATIC_ASSERT macro so we don't error-out on old C compilers.